controllers/controllers.go

```diff
@@ -34,14 +34,14 @@ var cookieStore = sessions.NewCookieStore([]byte("session-kesy"))
 
 // ここテストでstubするために関数ポインタをグローバル変数に代入しておきます．もしインスタンスメソッドではない関数をstubする方法があれば，書き換えて構わない．
 var CheckCSRFToken = checkCSRF
-var LoginRequired = checkLogin
+var LoginRequired = CheckLogin
```

Comment:
__[golint]__
* [golint] exported var LoginRequired should have comment or be unexported